analytics.js

```diff
@@ -1265,12 +1276,20 @@ if (isValidArgsArray) {
   let i = 0;
   while (i < argumentsArray.length) {
     if (argumentsArray[i] && argumentsArray[i][0] === defaultMethod) {
-      instance.toBeProcessedArray.push(argumentsArray[i]);
-      argumentsArray.splice(i, 1);
+      argumentsArray.unshift(argumentsArray.splice(i, 1)[0]);
       break;
     }
     i += 1;
   }
+
+  /**
+   * If defaultMethod is present in the buffered requests execute it.
+   */
+  if (argumentsArray.length > 0 && argumentsArray[0][0] === defaultMethod) {
+    argumentsArray[0].shift();
+    instance[defaultMethod](...argumentsArray[0]);
+    argumentsArray.shift();
+  }
```

Comment:
Let's not execute the `load` API separately

Reply:
Addressed